pep-0696.rst

```diff
@@ -346,6 +369,8 @@ subtype of one of the constraints.
    TypeVar("Ok", float, str, default=float)     # Valid
    TypeVar("Invalid", float, str, default=int)  # Invalid: expected one of float or str got int
 
+.. _function-defaults:
```

Comment:
Since every ref target across every PEP shares a namespace, to avoid collisions please prefix all ref target labels with the PEP number, i.e.

```rst
.. _696-function-defaults:
```

You'll also need to update their usage accordingly in `:ref:` roles.

If you only add the number to the existing ref target labels, as suggested above, it will be automatically stripped when generating the HTML fragment anchors, so it won't break any existing external links (and passing `-n` will generate a warning on any broken internal ones).